### PR TITLE
chore: capitalize wallet name in snackbar

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-6972

### CODE REVIEW

File: src/utils/capitalizeFirstLetter.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const capitalizeFirstLetter = (text: string | undefined) => {
+  return text !== undefined && text.charAt(0).toUpperCase() + text.slice(1);
```
Comment (DNR500) [inline src/utils/capitalizeFirstLetter.ts:2]: This looks fine as you want to do the capitalisation in the hook. Just be aware you can do this with CSS easily too and its often good to think about when its best to use either approach

```
&:first-letter{
  text-transform: capitalize
}
```

**Review by DNR500 — APPROVED** [review]

File: src/utils/capitalizeFirstLetter.ts (hunk 1 above)
Comment (0xsebn) [reply src/utils/capitalizeFirstLetter.ts:2]: Thats a good point. I agree that this approach is a bit better just for terms of simplicity.

**Review by 0xsebn — APPROVED** [review]
Looks like build is failing but besides that good to go. Either approach (the one mentioned by Nathan) or the one implemented looks good to me.

File: src/utils/capitalizeFirstLetter.ts (hunk 1 above)
Comment (dennyscode) [reply src/utils/capitalizeFirstLetter.ts]: Great point. I followed you advice!

**Pull request merged**